sdk/src/Services/DynamoDBv2/Custom/DataModel/Context.cs

```diff
@@ -208,7 +208,7 @@ public MultiTableBatchGet CreateMultiTableBatchGet(params BatchGet[] batches)
         /// </summary>
         /// <typeparam name="T">Type of objects to write</typeparam>
         /// <returns>Empty strongly-typed BatchWrite object</returns>
-        public BatchWrite<T> CreateBatchWrite<T>()
+        public IBatchWrite<T> CreateBatchWrite<T>()
```

Comment:
We can't change return types even to the interface because that could be a breaking change to existing applications that are treating the return as a `BatchWrite` variable.